src/components/SocialShare.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react';
+import { TwitterShareButton, FacebookShareButton, LinkedinShareButton } from 'react-share';
+import styled from 'styled-components';
+
+import FacebookIcon from '../assets/images/svgs/facebook.svg';
+import TwitterIcon from '../assets/images/svgs/twitter.svg';
+import LinkedinIcon from '../assets/images/svgs/linkedin.svg';
+
+const Social = props => (
+  <Fragment>
+    <Social.Icon
+      src={props.iconUrl}
+    />
```

Comment:
I think it will make sense to have an `alt` attribute here for accessibility purposes